README.md

```diff
@@ -1,6 +1,71 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;RAFT: RAPIDS Analytics Frameworks Toolset</div>
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;RAFT: RAPIDS Analytics Frameworks Toolkit</div>
```

Comment:
We need to change the word toolset to toolkit here too: https://github.com/rapidsai/raft/blob/c2daa5c97e9dcf6af850f9c04bcbabee914b4500/cpp/CMakeLists.txt#L157

Reply:
I'm okay w/ toolset or toolkit, really. For some reason, I personally find toolkit rolls off my tongue more easily. What do you think?

Reply:
I think toolkit is slightly nicer too, but I’m fine with either, just mentioning that we need consistency (which can be in another PR to keep this one focused on the README)